python/nano/client-application/StableDiffusion/stable_diffusion/pipelines/multi_instance.py

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+def multi_instance_generate(pipe, prompts, save_paths, result_queue, batch_size=4, **kwargs):
```

Comment:
I don't think we need this for the example

Reply:
Have removed multi_instance related code.